.env.sample

```diff
@@ -1,29 +1,27 @@
-#create environment file name as .env
-#and place following configuration data.
 
+#create environment file name as .env.local
+#and place following configuration data.
 
 VITE_CONTENTSTACK_API_KEY=your_stack_api_key
 VITE_CONTENTSTACK_DELIVERY_TOKEN=your_delivery_token
 VITE_CONTENTSTACK_ENVIRONMENT=your_environment_token
 
 # Below config options are for enabling live preview/live edit tags for the starter app
 
-VITE_CONTENTSTACK_API_HOST= for(NA: api.contentstack.io, EU: eu-api.contentstack.com)
-VITE_CONTENTSTACK_MANAGEMENT_TOKEN = your_management_token
+VITE_CONTENTSTACK_PREVIEW_HOST= rest-preview.contentstack.com
+VITE_CONTENTSTACK_PREVIEW_TOKEN= your_live_preview_token
 VITE_CONTENTSTACK_APP_HOST=app.contentstack.com
 VITE_CONTENTSTACK_LIVE_PREVIEW= true
 VITE_CONTENTSTACK_LIVE_EDIT_TAGS= false
 
+# VITE_CONTENTSTACK_API_HOST=api.contentstack.com
+# VITE_CONTENTSTACK_REGION=eu
```

Comment:
is this region value for default ? why is it eu and not us ?

Reply:
EU is an example value and Note is added at the bottom as 'us' is default value